VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -531,37 +522,27 @@ protected void hideSoftKeypad() {
     }
 
     public boolean showErrorMessage(String header, String message) {
-        return showErrorMessage(header, message, true);
-    }
+        hideErrorMessage();
+        alertDialog = new AlertDialog.Builder(this)
+                .setTitle(header)
+                .setMessage(message)
+                .setCancelable(false)
```

Comment:
Still don't think we should break the back button, how about using `setCanceledOnTouchOutside(false)` instead?